packages/netlify-cms-core/src/reducers/collections.js

```diff
@@ -8,7 +8,7 @@ import { formatToExtension } from 'Formats/formats';
 
 const collections = (state = null, action) => {
   switch (action.type) {
-    case CONFIG_SUCCESS:
+    case CONFIG_SUCCESS: {
       const configCollections = action.payload ? action.payload.get('collections') : List();
```

Comment:
Wrape the case in a block to prevent variables (e.g. `const`) from leaking to the upper scope.